lib/gettext/compiler.ex

```
@@ -29,6 +29,14 @@ defmodule Gettext.Compiler do
     opts = Keyword.merge(mix_config_opts, compile_time_opts)
 
     priv = Keyword.get(opts, :priv, @default_priv)
+    wildcard = Keyword.get(opts, :compiler_po_wildcard)
+    # NOTE: compiler_po_wildcard format: "gettext/*/LC_MESSAGES/*.po"
+    if wildcard do
+      [_ | wildcard] = Path.split(wildcard)
```

I don't think we should silently ignore the first directory. We should document that the compiler_po_wildcard is used from inside `priv`.